product_seeds/views/legal_information_certification.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
File name conventions from the [OCA](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming) would be: `legal_information_certification_views.xml` (I did not follow this convention until recently).